src/main/resources/config/liquibase/changelog/20220903100502_changelog.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:pro="http://www.liquibase.org/xml/ns/pro" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/pro http://www.liquibase.org/xml/ns/pro/liquibase-pro-3.8.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.8.xsd">
+    <changeSet author="ge65cer" id="20220903100502-1">
+        <comment>Fill the learning_goal_exercise table with relations currently stored in the learning_goal_lecture_unit table.</comment>
+        <sql>
+            INSERT INTO learning_goal_exercise (learning_goal_id, exercise_id)
+            SELECT lg.learning_goal_id, lu.exercise_id FROM learning_goal_lecture_unit lg
+            LEFT JOIN lecture_unit lu ON lu.id = lg.lecture_unit_id
+            WHERE lu.discriminator = "E";
+        </sql>
+    </changeSet>
+    <changeSet author="ge65cer" id="20220903100502-2">
+        <comment>Remove all relations between exercise units and learning goals from the learning_goal_lecture_unit table.</comment>
```

Comment:
💡Idea: Just to be extra safe here, you could have a precondition that the above changeSet has been really executed and we do not lose any data. I know the changeSets should be executed in order, but maybe just to be safe here :) 

See: https://docs.liquibase.com/concepts/changelogs/preconditions.html#:~:text=a%20given%20property.-,changeSetExecuted,-Defines%20if%20the

Reply:
I [had a pre-condition](https://github.com/ls1intum/Artemis/pull/5642/commits/0234f1d9cfe23859ad8c619ff101c747cead11e2) that the table already exists, but then the Cypress tests were failing to build.